controllers/grid/settings/user/form/UserDetailsForm.php

```diff
@@ -182,7 +182,7 @@ public function initData()
             $data = [
                 'givenName' => $author->getGivenName(null), // Localized
                 'familyName' => $author->getFamilyName(null), // Localized
-                'affiliation' => $author->getAffiliation(null), // Localized
```

Comment:
Hmmm... This file is used in the quickSubmit plugin. I do not think we can use the new affiliationField UI component there :-( I/we need to think how to solve it best...